ci-operator/step-registry/openshift/upgrade/aws-check-dev-symlinks/openshift-upgrade-aws-check-dev-symlinks-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-aws-check-dev-symlinks
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - chain: ipi-aws-pre-stableinitial
+    test:
+    - ref: storage-disk-symlinks-save
+    - ref: openshift-e2e-test
+    - ref: storage-disk-symlinks-check
+    post:
+    - ref: openshift-e2e-test-capabilities-check
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
+      TEST_TYPE: upgrade
+      TEST_SUITE: ""
+      TEST_UPGRADE_OPTIONS: ""
+      DELETE_MC: "false"
+  documentation: |-
+    The Openshift E2E AWS workflow executes the upgrade end-to-end test suite on AWS with a default cluster configuration checking that root device symlinks are in place after upgrade.
```

Comment:
Please add a note that the purpose is to check that udev rules are stable across upgrades.